docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -77,70 +85,107 @@ feedbackIntegration({
+| `background`              | `--background`                 | `#ffffff`                                 | `#29232f`                                 | Background color of the widget (injected button and form)                                             |
 | `border`                  | `--border`                     | `1.5px solid rgba(41, 35, 47, 0.13)`      | `1.5px solid rgba(235, 230, 239, 0.15)`   | The border style used for the widget (injected button and form)                                       |
-| `borderRadius`            | `--border-radius`              | `12px`                                    | `12px`                                    | Border radius style used for the widget (injected button and success message)                         |
 | `boxShadow`               | `--box-shadow`                 | `0px 4px 24px 0px rgba(43, 34, 51, 0.12)` | `0px 4px 24px 0px rgba(43, 34, 51, 0.12)` | The box shadow style used for the widget (injected button and form)                                   |
+
+| `inputForeground`         | `--input-foreground`           | `inherit`                                 | `inherit`                                 | Foreground color for form inputs                                                                      |
+| `inputBackground`         | `--input-background`           | `inherit`                                 | `inherit`                                 | Background color for form inputs                                                                      |
+| `inputBackgroundFocus`    | `--input-background-focus`     | `inherit`                                 | `inherit`                                 | Background color for form inputs when focused                                                         |
+| `inputBorder`             | `--input-border`               | `var(--border)`                           | `var(--border)`                           | Border styles for form inputs                                                                         |
+| `inputBorderRadius`       | `--input-border-radius`        | `6px`                                     | `6px`                                     | Border radius style for form content (for example: inputs, buttons)                                   |
+| `inputOutlineFocus`       | `--input-outline-focus`        | `rgba(108, 95, 199, 1)`                   | `rgba(108, 95, 199, 1)`                   | Outline color for form inputs when focused                                                            |
+
+| `buttonBackground`        | `--button-background`          | `var(--background)`                       | `var(--background)`                       | Background color for the cancel button                                                                |
```

Comment:
No longer just the cancel button w/ the addition of screenshot related buttons right?

Reply:
ah, missed a spot.